lib/features/school_event/screens/school_event_organisations_screen.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/exhibition_flow/widgets/organisation_item.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/features/profiles/widgets/coin_widget.dart';
+import 'package:givt_app_kids/features/recommendation/organisations/cubit/organisations_cubit.dart';
+import 'package:givt_app_kids/helpers/app_theme.dart';
+import 'package:givt_app_kids/helpers/snack_bar_helper.dart';
+import 'package:givt_app_kids/shared/widgets/givt_back_button.dart';
+import 'package:loading_animation_widget/loading_animation_widget.dart';
+
+class SchoolEventOrganisationsScreen extends StatelessWidget {
```

Comment:
What's the difference between this screen and the original organisationsscreen? I think it should be the same thing only the API url is different to get the organisations

Reply:
there is no much diff at the moment (since I didn't spect time on design part yet), but there will be at leats title text change and perhaps more. 
I'm trying to follow copy/paste approach in such 'disposable' cases/flows to make that code as much isolated as possible, and also to avoid possible errors after making 'main' screen more generic.
However I can delete it and use the 'original' one (from exhibition flow) for now if you like?